email-templates/user-lockout-en_US/user-lockout.html.ejs

```diff
@@ -0,0 +1,15 @@
+<% include ../common-en_US/header.ejs %>
+
+<h1>Hi <%=name%>,</h1>
+
+<p>We have noticed multiple failed attempts to login to your zen account so for security reasons have locked your account.  </p>
+
+<p>To unlock your account you will need to change your password.</p>
```

Comment:
"To unlock your account, follow the link below in order to change your password."
No need for the sentence before